docs/DeveloperGuide.adoc

```diff
@@ -64,13 +64,24 @@ image::LogicClassDiagram.png[]
 [discrete]
```

Comment:
The Parser is still named `AddressBookParser`, need to change!

Reply:
I agree with what @zhuhanming is pointing out.